docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,255 @@
+
+```Kotlin
+import mozilla.components.service.glean.Glean
+```
+
+Then inside of the `onCreate()` function of the `MainActivity`, add the following 2 lines to enable and initialize Glean:
+
+```Kotlin
+Glean.setUploadEnabled(true)
+Glean.initialize(this)
+```
+
+That’s it!  The app now has Glean installed and enabled and will be able to send the baseline ping without any additional work!
+
+### Add Custom Metric
+
+Since we are instrumenting crashes with some custom metrics, we still have some work to do.  We will need to add a
+`metrics.yaml` file to define the metrics we will use to record our crash information and a `pings.yaml` file to define a
+custom ping which will give us some control over the scheduling of the uploading of the crash telemetry.
+
+In order to do this we need to make a decision.  Mainly, what metric type will we use to represent our crash data.  This could
+probably be argued several ways, but I have chosen to instrument it as an [event](https://mozilla.github.io/glean/book/user/metrics/event.html),
+simply because events capture information in a nice concise way and have a built-in way of passing additional information
+using the `extras` field.  So if we want to pass along the cause, as well as a few lines of description or even potentially a
+few lines of stack trace, events let us do that easily (with [some limitations](https://mozilla.github.io/glean/book/user/metrics/event.html#limits)).
+
+Now that we have decided what metric type we will use, we can create our `metrics.yaml`.  Inside of the `app` folder in the
+Android Studio project we are going to create a new file and call it `metrics.yaml`.  Then we can add the schema definition and
+the metric definition to the file, so that it looks like this:
+
+```YAML
+# Required to indicate this is a `metrics.yaml` file
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+crash:
+  exception:
+    type: event
+    description: |
+      Event to record crashes caused by unhandled exceptions
+    notification_emails:
+      - crashes@example.com
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1582479
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1582479
+    expires:
+      2099-01-01
+    send_in_pings:
+      - crash
+    extra_keys:
+      cause:
+        description: The cause of the crash
+      message:
+        description: The exception message
+```
+
+By way of explanation, this creates a metric called `exception` within a metric category called `crash`.  There is a brief
+description and the required notification, bug, data review, and expiry fields.  Then we have the `send_in_pings` field with a
+value of `- crash`.  This means that we will send the crash data via a custom ping named `crash` (which we haven’t created,
+yet).  Finally we have the `extra_keys` field with two keys defined, `cause` and `message`.  This will allow us to send a
+couple of pieces of additional information along with the event.
```

Comment:
Worth mentioning that we require data review for all Mozilla applications.
See **Important** box on https://mozilla.github.io/glean/book/user/adding-new-metrics.html